### PR TITLE
add core benchmark

### PR DESCRIPTION
**Pull request by kmaehashi**

### CODE REVIEW

File: benchmarks/cupy/bench_core.py [hunk 1]
```diff
@@ -0,0 +1,20 @@
+from benchmarks.numpy.common import Benchmark
+from benchmarks.utils import sync
+from benchmarks.utils import parameterize
+
+import cupy
+import numpy
+
+
+@sync
+@parameterize([
```
Comment (asi1024) [inline benchmarks/cupy/bench_core.py:10]: How about parameterizing dtypes?

File: benchmarks/cupy/bench_core.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+    ('ndim', [1, 2, 5, 8]),
```
Comment (asi1024) [inline benchmarks/cupy/bench_core.py]: How about adding the case for ndim = 0?

Comment (kmaehashi) [reply benchmarks/cupy/bench_core.py]: Added.

File: benchmarks/cupy/bench_core.py (hunk 1 above)
Comment (kmaehashi) [reply benchmarks/cupy/bench_core.py:10]: Ok. I only added `['float32', 'complex128']` as there should be no much difference of time between dtypes.

**Comment by asi1024** [comment]
LGTM.

**Pull request merged**